packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -368,7 +363,26 @@ function UntranslatedComplexItem(props) {
 					>
 						{decoratedLength}
 					</sup>
-				</TooltipTrigger>
+				) : (
+					<TooltipTrigger
+						key="badge-tooltip"
+						className="offset"
+						label={getDataAbstract(data)}
+						tooltipPlacement="right"
+					>
+						<sup
+							key="badge"
+							className={`${theme.badge} badge`}
+							aria-label={t('TC_OBJECT_VIEWER_NB_CHILD', {
+								defaultValue: 'Contains {{count}} child object',
+								defaultValue_plural: 'Contains {{count}} child objects',
+								count: info.length,
+							})}
+						>
+							{decoratedLength}
+						</sup>
```

Comment:
please factorize it to not copy pasate those 10 lines

Reply:
Thanks, fixed it.